google/auth/_cloud_sdk.py

```diff
@@ -166,3 +103,27 @@ def load_authorized_user_credentials(info):
         token_uri=_GOOGLE_OAUTH2_TOKEN_ENDPOINT,
         client_id=info['client_id'],
         client_secret=info['client_secret'])
+
+
+def get_project_id():
+    """Gets the project ID from the Cloud SDK.
+
+    Returns:
+        Optional[str]: The project ID.
+    """
+
+    try:
+        output = subprocess.check_output([
+            'gcloud', 'config', 'config-helper', '--format', 'json'])
+    except (subprocess.CalledProcessError, OSError, IOError):
```

Comment:
It's a quirk between windows / unix. Ran into the same thing with Nox.

In general, it's better for this method to err on the side of being too broad in error handling than to potentially explode just because it can't talk to gcloud and figure out the project.